packages/scripts/src/getConversationText.ts

```diff
@@ -0,0 +1,41 @@
+import { MongoClient, BSON } from "mongodb";
+import { assertEnvVars } from "mongodb-rag-core";
+import { Conversation } from "mongodb-chatbot-server";
+
+import "dotenv/config";
+
+const { MONGODB_DATABASE_NAME, MONGODB_CONNECTION_URI } = assertEnvVars({
+  MONGODB_DATABASE_NAME: "",
+  MONGODB_CONNECTION_URI: "",
+});
+
+const conversationIdString = process.argv[2];
+if (!conversationIdString) {
+  throw new Error(
+    "Missing conversation id. Usage: getConversationText <conversationId>"
+  );
+}
+
+const conversationId = new BSON.ObjectId(conversationIdString);
+
+(async function main() {
```

Comment:
i have a personal aversion to IIFEs that i don't think is based on any particular reason. consider refactoring to:

```ts
async function main() {
  // do stuff
}
main()
```

Reply:
Sure why not